cli/index.js

```diff
@@ -0,0 +1,98 @@
+#!/usr/bin/env node
+
+const fs = require('fs-extra');
+const path = require('path');
+
+const run = require('./run');
+const copyFeatures = require('./copyFeatures');
+const replaceDredd = require('./replaceDredd');
+
+
+PROJECT_DIR = process.cwd();
+NODE_MODULES_DIR = path.join(PROJECT_DIR, 'node_modules');
+
+FEATURES_SRC_DIR = path.join(NODE_MODULES_DIR, 'dredd-hooks-template', 'features');
+FEATURES_DIR = path.join(PROJECT_DIR, 'features');
+STEPS_DIR = path.join(FEATURES_SRC_DIR, 'support');
+
+NODE_BIN = process.execPath;
+DREDD_BIN = path.join(NODE_MODULES_DIR, '.bin', 'dredd');
+CUCUMBER_BIN = path.join(NODE_MODULES_DIR, '.bin', 'cucumber-js');
+
+
+function init() {
+  // create a 'features' directory in the project we're initializing
+  fs.mkdirSync(FEATURES_DIR);
```

Comment:
What would happen if I run `init` on already initialized template/project?

Reply:
This throws, because the folder is going to already exist.

Reply:
Do you think I should make it a nicer user-facing error? I don't want to overcomplicate the CLI much and it's aimed at hooks handlers devs, so I expect them to be able to read uncaught exceptions quite fluently, even though they don't know JavaScript much.